src/plugins/dialog/math.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+
+import dialog from '../modules/dialog';
+
+const katexJsSrc = 'https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/katex.min.js';
+const katexCssSrc = 'https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/katex.min.css';
+
+const script = document.createElement('script');
+script.type = 'text/javascript';
+script.src = katexJsSrc;
+
+const link = document.createElement('link');
+link.rel = 'stylesheet';
+link.href = katexCssSrc;
+
+document.head.appendChild(link);
+document.head.appendChild(script);
+
+const htmlMathString = function (exp) {
+    return window.katex.renderToString(exp, {
+        throwOnError: false,
+    });
+}
```

Comment:
Code outside of "export default" is I'll modify after the merge.
It will modify it with the same usage as [CodeMirror](https://github.com/JiHong88/SunEditor#use-codemirror).